Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1

```diff
@@ -61,7 +61,7 @@ function Get-ResourceWinRmConfig
     $winrmPortToUse = ''
     $protocolToUse = ''
 
-    $environment = Get-Environment -environmentName $environmentName
+    $environment = Get-Environment -environmentName $environmentName -Connection $connection
```

Comment:
suggest to make this param mandatory field. this will ensure we don't miss in later stages